src/main/resources/Macros/MapMacro/Code/LeafletMap.xml

```diff
@@ -136,25 +136,26 @@
       var mapId = this.options.mapId !== undefined ? this.options.mapId : 'map-id';
       var tiles = this.options.tiles;
       var locationType = this.options.locationType ? this.options.locationType.toLowerCase() : 'address';
+      var info = this.options.info !== undefined ? this.options.info : '';
```

Comment:
I'm not sure of the interest of not leaving the `info` variable undefined, rather than affecting an empty value to it, what do you think? Isn't it simpler (less code) to leave it undefined? Also, in case we need to keep the `''` value, I think the following expression is preferrable, is it not? `var info = this.options.info || '';`

Reply:
I agree, undefined for the variable is fine. We can check if it is undefined instead of checking if its empty.